mkContainer/mkContainer.nix

```diff
@@ -147,10 +147,13 @@ let
         #   podman run -i [SIC]
         #     -> launches crippled interactive shell with no controlling
         #        terminal .. kinda useless
-        Entrypoint = [ "${environment}/activate" ];
+        Entrypoint = [
+          "${environment}/activate"
+          "--env"
+          environment
```

Comment:
question nonblocking: do you think it's worth adding description and cache as followup?

Reply:
Yeah, should have mentioned; I'm not sure off-hand how to pass them over and this PR is already getting very long.